load.php

```diff
@@ -2,182 +2,92 @@
-
-	// cache must be initted once it's included, else we'll get a fatal.
-	wp_cache_init();
-}
+require_once ROOT_DIR . '/includes/manifest.php';
 
 /**
- * Load the advanced-cache dropin.
+ * Retrieve plugin version from package.json.
  *
- * @param  bool $should_load
- * @return bool
+ * @return string
  */
-function load_advanced_cache( $should_load ) {
-	$config = get_config();
-
-	if ( ! $should_load || ! $config['batcache'] ) {
-		return $should_load;
-	}
-
-	require __DIR__ . '/dropins/batcache/advanced-cache.php';
+function version() {
+	return json_decode( file_get_contents( __DIR__ . '/package.json' ) )->version;
 }
 
 /**
- * Load the db dropin.
+ * Retrieve plugin docs version from package.json.
+ *
+ * @return string
  */
-function load_db() {
-	$config = get_config();
-
-	if ( ! $config['ludicrousdb'] ) {
-		return;
-	}
-
-	if ( ! defined( 'DB_CONFIG_FILE' ) ) {
-		define( 'DB_CONFIG_FILE', __DIR__ . '/dropins/db-config.php' );
-	}
-
-	require __DIR__ . '/dropins/ludicrousdb/ludicrousdb.php';
+function docs_version() {
+	return json_decode( file_get_contents( __DIR__ . '/package.json' ) )->docsVersion;
 }
 
 /**
- * Get available platform plugins.
+ * Get the docs site home URL.
  *
- * @return array Map of plugin ID => path relative to plugins directory.
+ * @return string
  */
-function get_available_plugins() {
-	return array(
-		's3-uploads'      => 's3-uploads/s3-uploads.php',
-		'aws-ses-wp-mail' => 'aws-ses-wp-mail/aws-ses-wp-mail.php',
-		'tachyon'         => 'tachyon/tachyon.php',
-		'cavalcade'       => 'cavalcade/plugin.php',
-		'redis'           => 'wp-redis/wp-redis.php',
-	);
+function docs_url() {
+	return defined( 'HM_DOCS_HOME' ) ? HM_DOCS_HOME : 'https://docs.aws.hmn.md/';
 }
 
-/**
- * Load the plugins in hm-platform.
- */
-function load_plugins() {
-	$config = get_config();
-
-	add_filter( 'plugins_url', function ( $url, $path, $plugin ) {
-		if ( strpos( $plugin, __DIR__ ) === false ) {
-			return $url;
-		}
-
-		return str_replace( WP_CONTENT_DIR, WP_CONTENT_URL, dirname( $plugin ) ) . $path;
-	}, 10, 3 );
-
-	// Force DISABLE_WP_CRON for Cavalcade.
-	if ( $config['cavalcade'] && ! defined( 'DISABLE_WP_CRON' ) ) {
-		define( 'DISABLE_WP_CRON', true );
+// Fix plugins URL for plugins in HM Platform.
+add_filter( 'plugins_url', function ( $url, $path, $plugin ) {
+	if ( strpos( $plugin, ROOT_DIR ) === false ) {
+		return $url;
 	}
 
-	foreach ( get_available_plugins() as $plugin => $file ) {
-		if ( ! $config[ $plugin ] ) {
-			continue;
-		}
+	return trailingslashit( str_replace( WP_CONTENT_DIR, WP_CONTENT_URL, dirname( $plugin ) ) ) . ltrim( $path, '/' );
+}, 10, 3 );
 
-		require __DIR__ . '/plugins/' . $file;
-	}
-
-	if ( ! empty( $config['elasticsearch'] ) ) {
-		require_once __DIR__ . '/lib/elasticpress-integration.php';
-		ElasticPress_Integration\bootstrap();
-	}
+/**
+ * Load the plugins.
+ */
+try {
+	Plugins\load_enabled_plugins();
+} catch ( Exception $exception ) {
+	die( $exception->getMessage() );
```

Comment:
For the love of everything, please no - are we _trying_ to make peoples sites unavailable?

Reply:
😁 intended to be caught on local but will remove it.